### ISSUE TITLE
Idle Notifier idle timer is too long

### ISSUE THREAD

**Issue by KeyboardOSRS** [runelite/runelite#8246]
On game tick, the Idle Notifier plugin checks if you have clicked in the last second to determine if you are idle. If you have clicked in the last second, all notifications for that tick are cancelled.

This is a problem with certain tasks such as pickpocketing with the Hitpoints Notification - if you spam click in intervals less than 1 second, the notification will never trigger unless you don't click for 1 second.

Prior to a commit on 2018-11-25, this timer was 200ms (adequate for pickpocketing). I'm not sure of the reasoning of increasing it to 1000ms, but it has affected pickpocketing.

**Comment by whartd** [comment]
Should the hit points notifier be handled differently? Perhaps notifying you of your hit points when thieving belongs somewhere other than the "idle" notifiers? A plug in implementing this could also show your pickpockets/hr and fail rate, something I think may have been mentioned in the past.

**Comment by xDemoN** [comment]
Spam-clicking in intervals less than 1 second isn't an "idle animation" - I'm of the opinion that this isn't a regression at all because this isn't an intended use-case for the idle notifier (You're the first person to point this out in 4 months)

**Comment by Twiglet1022** [comment]
> Prior to a commit on 2018-11-25, this timer was 200ms (adequate for pickpocketing). I'm not sure of the reasoning of increasing it to 1000ms, but it has affected pickpocketing.

I agree with what whatrd and xDemoN have said, but since you you're wondering, that was my commit. The reason for that is in the description of this video demo that I attached to the PR: https://www.youtube.com/watch?v=sQC_PRidlo0

**Comment by rsekelsky** [comment]
Bumping this, I also agree with the above comments. Best course of action is to separate the HP/prayer notifiers from the idle notifier and restore the 200ms timer (perhaps less, could be 100ms).
